Makefile

```diff
@@ -0,0 +1,20 @@
+VENV_NAME=.venv
+PYTHON=${VENV_NAME}/bin/python3
+PIP=${VENV_NAME}/bin/pip
+
+.PHONY: venv
+venv:
+	if [ ! -d "${VENV_NAME}" ]; then python3 -m venv ${VENV_NAME}; fi
+
+.PHONY: install
+install: venv
+	@ ${PIP} install -r requirements.txt && \
```

Comment:
I think `@` can be removed so it's clear what's being executed. Trying to learn `Makefile`, feel free to disagree and ignore this.

Reply:
totally can do! Eventually, you'll want the `@` but I'll wait for your eyes to stop bleeding from makefile syntax first 😛